index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
```

Comment:
Please stick to the given Figma Design as much as possible: 😃✔
 
If this is the Figma template you are using : 

![image](https://user-images.githubusercontent.com/87448628/164540405-b1420cca-16b7-476b-aec1-506c254f2325.png)

You still need to add the social media links at the bottom of your page and the background images highlighted in the image above:

![image](https://user-images.githubusercontent.com/87448628/164540187-9921e594-87c4-4f83-8368-3730449f3ac0.png)

here is a link to Figma templates:  [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
Kindly fix this issue ✔👍

Reply:
I have added all suggested patches and now moving to request another review.